src/strategyHelpers/blueGreen/blueGreenHelper.test.ts

```diff
@@ -1,20 +1,25 @@
 import {
-   createWorkloadsWithLabel,
-   deleteWorkloadsAndServicesWithLabel,
+   deployWithLabel,
+   deleteGreenObjects,
+   fetchResource,
+   getDeploymentMatchLabels,
    getManifestObjects,
    getNewBlueGreenObject,
    GREEN_LABEL_VALUE,
-   isServiceRouted,
-   NONE_LABEL_VALUE
+   isServiceRouted
 } from './blueGreenHelper'
 import * as bgHelper from './blueGreenHelper'
 import {Kubectl} from '../../types/kubectl'
 import * as fileHelper from '../../utilities/fileUtils'
+import {K8sObject} from '../../types/k8sObject'
+import * as manifestUpdateUtils from '../../utilities/manifestUpdateUtils'
+import {ExecOutput} from '@actions/exec'
 
 jest.mock('../../types/kubectl')
+let testObjects
```

Comment:
move this inside the test scope (the describe). You should probably also reset its value on beforeEach. 

Limit scope as much as possible to prevent future errors